runtime/altair/src/lib.rs

```diff
@@ -1377,6 +1392,35 @@ impl pallet_liquidity_pools_gateway::Config for Runtime {
 	type WeightInfo = ();
 }
 
+#[cfg(feature = "testnet-runtime")]
+impl pallet_liquidity_rewards::Config for Runtime {
+	type AdminOrigin = EnsureRootOr<HalfOfCouncil>;
```

Comment:
One of the reasons it was not still in `altair` & `centrifuge` was to choose this value properly.

All functions to configure reward groups should be relatively fast and done by some kind of administrator (maybe). Those are: increasing the distributed reward, setting epochs, creating groups, etc.

Ideally, these functions will be handled by governance, but maybe the governance process is too slow for this. i.e. If a new pool is created with N tranches, we should configure then some rewards groups or associate those tranches to an existing group. Doing by governance would imply some period where the reward system is not still working for that pool.

Reply:
Couldn't we do it as part of the pool creation process?

Reply:
I think it could work. Just a question:
- Would adding a pool also increase the total distributed reward or just would take some amount of the current distribution reward?

Reply:
Once a pool is created, from liquidity reward PoV:
- It would require (or not) to create some new group for some new % rate. All groups could be created at the beginning.
- For each tranche, you need to move each PoolXTrancheX currency into the reward group with the corresponding %.
- Decide if we need to increase the minted reward (I think this can be done always in another extrinsic handle by economics)

Reply:
For now I think this needs to be governance. I would imagine what will happen initially is that soon the first 1 or 2 pools will launch, and then there will be a first governance proposal that sets the rate, creates the groups, and assigns the existing tranches from those 1 or 2 pools to those reward groups. Then afterwards it definitely makes sense to batch this with the pool creation process, but the reward groups need to be known for this.

> Would adding a pool also increase the total distributed reward or just would take some amount of the current distribution reward? I would say the second choice, because in the creation moment, the stake remains the same

Yes the total distributed reward should not change with more pools.